### ISSUE TITLE
panic generating AAT

### ISSUE THREAD

**Issue by edjroz** [pokt-network/pocket-core#1019]
**Describe the bug**
Creating an AAT using `pocket apps create-aat <addr> <clientPublicKey>`

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x19f2170]

goroutine 1 [running]:
github.com/pokt-network/pocket-core/app/cmd/cli.glob..func21(0x26930c0, 0xc00036eea0, 0x2, 0x2)
        /Users/edjroz/Repos/pocket-core/app/cmd/cli/app.go:141 +0x430
github.com/spf13/cobra.(*Command).execute(0x26930c0, 0xc00036ee60, 0x2, 0x2, 0x26930c0, 0xc00036ee60)
        /Users/edjroz/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:846 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0x2698f40, 0x10074df, 0xc000098058, 0x0)
        /Users/edjroz/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/edjroz/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
github.com/pokt-network/pocket-core/app/cmd/cli.Execute()
        /Users/edjroz/Repos/pocket-core/app/cmd/cli/root.go:39 +0x2d
main.main()
        /Users/edjroz/Repos/pocket-core/app/cmd/pocket_core/main.go:8 +0x20
```

**Comment by oten91** [comment]
app.PCA is null at the moment of the call

```
aatBytes, err := app.PCA.GenerateAAT(hex.EncodeToString(kp.PublicKey.RawBytes()), args[1], privkey)
if err != nil {
 fmt.Println(err)
 return
}
````

**Comment by andrewnguyen22** [comment]
closed with #1014
